src/sentry/static/sentry/app/utils/eventWaiter.tsx

```diff
@@ -109,14 +109,14 @@ class EventWaiter extends React.Component<Props, State> {
       firstIssue = issues.find((issue: Group) => issue.firstSeen === firstEvent) || true;
 
       recordAnalyticsFirstEvent({
-        key: 'first_event_recieved',
+        key: 'first_event_received',
```

Comment:
Cannot be changed until this is changed https://github.com/getsentry/reload/blob/67d393e63089a6044c499e3795fae8d169425e5a/reload_app/events.py#L663